staging/src/k8s.io/client-go/tools/leaderelection/healthzadaptor_test.go

```diff
@@ -53,6 +54,11 @@ func (fl *fakeLock) Identity() string {
 	return fl.identity
 }
 
+// RawRecord is a dummpy to allow us to have a fakeLock for testing.
```

Comment:
This is no longer needed - please remove

Reply:
Addressed.